### PR TITLE
README: suggest use of line folding

### PR DESCRIPTION
**Pull request by Cireo**
The `>-` operator is convenient and quickly becomes more readable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ example:
 
 ## Ignore paths and names
 
-You can use the `ignore_paths` and `ignore_names` input to disable specific directories and files.
+You can use the `ignore_paths` and `ignore_names` input to disable specific directories and files. These are passed as environment variables, and should evaluate to a single space separated string. It may be convenient to use [`>-`](https://yaml.org/spec/1.2.2/#65-line-folding) for readability if you have multiple selectors.
```
Comment (port19x) [inline README.md]:
```suggestion
You can use the `ignore_paths` and `ignore_names` input to disable specific directories and files.
These are passed as environment variables, and should evaluate to a single space separated string. 
It may be convenient to use [`>-`](https://yaml.org/spec/1.2.2/#65-line-folding) for readability if you have multiple selectors.
```

File: README.md [hunk 2]
```diff
@@ -174,5 +176,5 @@ If running the latest stable version of Shellcheck is not to your liking, you ca
    - name: Run ShellCheck
      uses: ludeeus/action-shellcheck@master
      with:
-       version: v0.7.0
+       version: v0.9.0
```
Comment (port19x) [inline README.md]: Neither of those versions seem available as [github releases](https://github.com/ludeeus/action-shellcheck/releases)

Comment (port19x) [reply README.md]: How about 1.0.0?
since v1.1.0 is latest, that would be the next viable candidate

**Review by port19x — APPROVED** [review]
Looks good overall

File: README.md (hunk 2 above)
Comment (Cireo) [reply README.md]: I chose this per the README
> When specifying a custom version, please use any of the released versions listed in the [Shellcheck repository](https://github.com/koalaman/shellcheck/tags).

The version is presumably for `shellcheck`, not for `action-shellcheck`

File: README.md (hunk 1 above)
Comment (Cireo) [reply README.md]: Sure, incorporated.

**Comment by Cireo** [comment]
Note the checks won't run without someone poking them

Screen Shot 2023-01-15 at 11 01 00 AM

File: README.md [hunk 3]
```diff
@@ -174,5 +178,5 @@ If running the latest stable version of Shellcheck is not to your liking, you ca
...
```
Comment (ludeeus) [inline README.md]:
```suggestion
       version: v0.7.0
```
Unrelated change

File: README.md (hunk 2 above)
Comment (ludeeus) [reply README.md]: This is indeed for the binary, but its also not related to this.

**Review by ludeeus — APPROVED** [review]
Thanks 👍
